corehq/toggles.py

```diff
@@ -989,6 +989,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_USER]
 )
 
+ICDS_DASHBOARD_SHOW_MOBILE_APK = DynamicallyPredictablyRandomToggle(
+    'icds_dashboard_show_mobile_apk',
+    'Show a "Mobile APK" download link on the ICDS Dashboard',
```

Comment:
just for my knowledge and might be completely unrelated. This mobile APK is the android app which renders the mobile dashboard views right?

Reply:
Yep, exactly